elements/rh-secondary-nav/rh-secondary-nav-menu.ts

```diff
@@ -0,0 +1,163 @@
+import { html, LitElement } from 'lit';
+import { customElement, property, queryAssignedNodes, query } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+
+import { pfelement, observed } from '@patternfly/pfe-core/decorators.js';
+import { getRandomId } from '@patternfly/pfe-core/functions/random.js';
+
+import { tabletLandscapeBreakpoint } from '../../lib/tokens.js';
+import { MatchMediaController } from '../../lib/MatchMediaController.js';
+
+import { RhSecondaryNavDropdown } from './rh-secondary-nav-dropdown';
+
+import styles from './rh-secondary-nav-menu.css';
+
+/**
+ * @summary 'A pop up menu for secondary nav, available in full-width and fixed-with sizes'
+ *
+ * @slot base             - Slot to override all shadow dom contents for full-width or fixed-width menus
+ * @slot section          - Hidden if empty, section slot applies an auto column grid system for `<ul>` link lists
+ * @slot links            - Only for type="fixed-width", expects a `<ul>`
+ * @slot cta              - Menu level CTA, expects a `<pfe-cta>`
+ *
+ * @csspart full-width    - {HTMLElement} container - The wrapper for full-width menus
```

Comment:
```suggestion
 * @csspart full-width    - container - The wrapper for full-width menus
```
and others. `@csspart` doesn't take a type, the analyzer will just spit out a description string with `{HTMLElement}` in it, which isn't great.

Reply:
Fixed in 2f88b975241399448e917d6be39fb408e4026d29